<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicts Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddFolder">
       <property name="text">
        <string>Add Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveFolder">
       <property name="text">
        <string>Remove Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetFolders"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkUseFilename">
       <property name="text">
        <string>Use filename as dict label</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkExportMedia">
       <property name="text">
        <string>Export media files</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddFolder</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>add_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveFolder</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>remove_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_folder()</slot>
  <slot>remove_folder()</slot>
 </slots>
</ui>
